src/m365/teams/commands/app/app-update.ts

```diff
@@ -25,32 +26,76 @@ class TeamsAppUpdateCommand extends GraphCommand {
   public get description(): string {
     return 'Updates Teams app in the organization\'s app catalog';
   }
+  
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = typeof args.options.id !== 'undefined';
+    telemetryProps.name = typeof args.options.name !== 'undefined';
+    return telemetryProps;
+  }
 
   public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
-    const { id: appId, filePath } = args.options;
+    const { filePath } = args.options;
+
+    this
+      .getAppId(args)
+      .then((appId: string): Promise<any> => {
```

Comment:
Since we're not expecting any output, we shouldn't use `Promise<any>` but `Promise<void>`

Reply:
I fixed it